README.md

```diff
@@ -17,6 +17,15 @@ If you configure your local DNS server to include the Golink server as `go`, the
 [http://go/example](http://go/example) directly in your browser without needing to provide the fully qualified domain
  name (FQDN) of the server.
 
+## Usage
+
+```bash
+python3 -m venv venv
+source venv/bin/activate
+pip3 install --upgrade pip
```

Comment:
I'm not sure if upgrading `pip` is strictly necessary if you've just created a virtual-env. Was there a specific need for it?

Reply:
Removed. No, it just happened that my system pip was out of date, but I don't think there's anything that requires an absolutely latest pip.